### ISSUE TITLE
Azure Hosting model

### ISSUE THREAD

**Issue by riccardobecker** [chronoxor/NetCoreServer#55]
Hi there,
first of all, thanks for NetCoreServer, great product.

What is your opinion on how to host a "NetCoreServer" on Azure? AppService? How to put multiple instances of the server?

I am curious about your opinion on this

regars,
Riccardo

**Comment by chronoxor** [comment]
Yes just host one or more servers in your AppService, bind them to different ports and start them:
```
var content = new HttpsServer(context, IPAddress.Any, 8443);
var chat = new ChatServer(context, IPAddress.Any, 8444);

content.Start();
chat.Start();
```

**Comment by riccardobecker** [comment]
Well, yes it's that easy on startup of the appservice true. Some questions:

Did you already run it on Azure? 
How did you benchmark it?
How would you compare the performance against the aspnetcore default websocket implementation?

thanks!
